.github/workflows/cd-server.yml

```diff
@@ -0,0 +1,87 @@
+name: Deploy server
+on:
+  push:
+    branches:
+      - dev
+    tags:
+      - "[v]?[0-9]+.[0-9]+.[0-9]+*"
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        default: "enable"
```

Comment:
@ntampakas may i know why this default value is 'enable'?

Reply:
My bad @yuroitaki 
This is just a typo. Thanks for noticing it!
Fixed: https://github.com/ntampakas/tlsn/commit/56c4aa5074f9155fd9550bbde611348053c6a64b